contrib/experimental/great_expectations_experimental/expectations/expect_multicolumn_sum_values_to_be_equal_to_single_column.py

```diff
@@ -89,6 +90,8 @@ class ExpectMulticolumnSumValuesToBeEqualToSingleColumn(MulticolumnMapExpectatio
     """
     # </snippet>
 
+    additional_value: Optional[int] = None
```

Comment:
I see examples of floats for column values. This should probably allow floats as well, right?